apps/docs/src/app/docs/architecture/page.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Architecture
+nextjs:
+  metadata:
+    title: Architecture
+    description: Overview of the Blobscan's architecture
+---
+
+Blobscan is composed of different components that talk to each other over HTTP through REST endpoints and RPCs.
+
+## Diagram
+
+{% figure  src="/architecture.svg" appendCurrentTheme=true /%}
+
+## Overview
+
+Blobscan is comprised of the following major components:
```

Comment:
Since you mention Vercel, I would make a difference here mentioning Blobscan.com instead of Blobscan